autogpt_platform/frontend/src/components/agptui/Navbar.stories.tsx

```diff
@@ -19,14 +20,14 @@ const mockCreditData = {
 };
 
 // Mock the API module
-jest.mock("@/lib/autogpt-server-api", () => {
-  return function () {
-    return {
-      getStoreProfile: () => Promise.resolve(mockProfileData),
-      getUserCredit: () => Promise.resolve(mockCreditData),
-    };
-  };
-});
+// jest.mock("@/lib/autogpt-server-api", () => {
+//   return function () {
+//     return {
+//       getStoreProfile: () => Promise.resolve(mockProfileData),
+//       getUserCredit: () => Promise.resolve(mockCreditData),
+//     };
+//   };
+// });
```

Comment:
@Swiftyos not sure what to do about this one.. it will fail its tests once renabled but we can't mock with jest like this currently. Any input here would be appreciated